pkg/k8sclient/clientset/versioned/fake/empty_test.go

```diff
@@ -1,6 +0,0 @@
-// This is an empty test file used to force Go test to include this package in
```

Comment:
Weren't these added to make the test reporting accurate?

Reply:
Yes, but they were added even to directories that shouldn't need to be tested, such as directory which only contains generated code.

https://github.com/cyberark/secretless-broker/blob/92983c5fbd0fcb992c340954efb694afc00b64a9/pkg/k8sclient/clientset/versioned/fake/doc.go#L17